Python/clinic/_warnings.c.h

```diff
@@ -7,7 +7,9 @@ preserve
 #  include "pycore_runtime.h"       // _Py_ID()
 #endif
 
-
+#define WARNINGS_FILTERS_MUTATED__METHODDEF    \
+    {"_filters_mutated", _PyCFunction_CAST(warnings_filters_mutated), METH_NOARGS, NULL},
+    
```

Comment:
This is a generated file, it can't be edited directly.  `warnings_filters_mutated` would need to be converted to Argument Clinic, though it looks like it should be a straightforward one to do.

Reply:
opps, my bad, will do the needful.